src/main/java/se/iths/springbootgroupproject/dto/CreateMessageFormData.java

```diff
@@ -0,0 +1,40 @@
+package se.iths.springbootgroupproject.dto;
+
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import se.iths.springbootgroupproject.entities.Message;
+import se.iths.springbootgroupproject.entities.User;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class CreateMessageFormData {
+
+    @NotNull
+    @NotBlank
+    @Size(min = 1, max = 100, message = "Title must be between 1 and 100 chars")
+    private String title;
+    @NotBlank
+    @NotNull
+    @Size(min = 1, message = "Do not leave empty")
```

Comment:
I get Internal Server Error, status=500 if my message is longer than 255 char. 
It's because we have message set to VARCHAR(255) in db.
Add 'max = 255' and a proper message and it will work fine!

Reply:
Did not think of that at all, great job!